task_DEV-7/Input.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace TriangleAnalysis
+{
+    class Input
+    {
+        private string[] args;
+
+        public const string REPEAT_ENTRY = "Please, repeat entry.\n";
+        public const string ENTER_TRIANGLE_SIDES = "Enter lengths of triangle sides (LTS), dividing numbers by spaces:";
+        public const string CL_ARGUMENTS_LENGTH = "The number of command-line arguments is not equal to 3.";
+        public const string SIDES_LENGTHS_NUMBER = "The number of LTS is not equal to 3.";
+
+        public Input(string[] args)
+        {
+            this.args = args;
+        }
+
+        public string[] EnterTriangleSides()
+        {
+            string[] triangleSides = null;
+            bool repeat = false;
+            do
+            {
+                try
+                {
+                    if (args.Length == 0 || repeat == true)
+                    {
+
+                        triangleSides = EnterTriangleSidesConsole();
+                    }
+                    else
+                    {
+                        triangleSides = EnterTriangleSidesCommandLine(args);
+                    }
+                    TriangleSidesValidator validator = new TriangleSidesValidator(triangleSides);
+                    validator.CheckValidity();
+                    repeat = false;
+                }
+                catch (FormatException e)
```

Comment:
No need to catch each type of possible exception, the processing is similar to all, use only one catch